### PR TITLE
fix: treasure

### PR DESCRIPTION
**Pull request by vinckr**
Finally adds MC Treasure Irie to residents and tweaks `about` text a bit (more emoji & removing inexistent telegram+some more)

we kinda need better pictures for the other residents!

### CODE REVIEW

File: src/app/about/page.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ const About = () => {
               Eisbach Callin is a underground dance party at home in Munich,
               Germany since 2010. All of this is only possible through the
               dedication and love of all of our supporters. Hugs and kisses for
-              your support over the years! 🤗😚
+              your support over the years! 🤗🤗😚😚
```
Comment (jfcurran) [inline src/app/about/page.tsx:25]: 🚀 😍  lesgo

File: src/app/about/page.tsx [hunk 2]
```diff
@@ -62,7 +53,7 @@ const About = () => {
               >
                 mixcloud
               </Link>
-              .
+              ! We don't record much, but when we do, it's usually pretty good.
```
Comment (jfcurran) [inline src/app/about/page.tsx:56]: 🔥 🚒  call the fire department

**Review by jfcurran — APPROVED** [review]
LGTM

**Comment by jfcurran** [comment]
from build 
```
10:33:30 PM: 56:23  Error: `"` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities
10:33:30 PM: 56:57  Error: `"` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities
```

File: src/config/residents.ts [hunk 3]
```diff
@@ -55,6 +55,18 @@ const residents: Resident[] = [
       },
     ],
   },
+  {
+    name: 'MC Treasure Irie',
+    avatar: '/artists/treasureirie.jpg',
+    description:
+    "MC TREASURE IRIE is a DJ, lead singer, and MC. He exclusively sings his own songs and lyrics, which revolve around social criticism, politricks, and life's controversies. MC TREASURE IRIE originally comes from Ecuador and can now look back on a 20-year stage career. The word acrobat started with a hip-hop band in his hometown of Memmingen. After various reggae bands and sound projects in dub, hip-hop & drum'n'bass, MC TREASURE IRIE has been active on the mic for the Dandelion Soundsystem for over 10 years, as well as with Eisbach Callin', Siren Sisters, at Rootsbase/Fusion Festival, and the Siko Munich. He writes his lyrics in English, patois slang, as well as in German.",
```
Comment (jfcurran) [inline src/config/residents.ts:62]:
```suggestion
    'MC TREASURE IRIE is a DJ, lead singer, and MC. He exclusively sings his own songs and lyrics, which revolve around social criticism, politricks, and life’s controversies. MC TREASURE IRIE originally comes from Ecuador and can now look back on a 20-year stage career. The word acrobat started with a hip-hop band in his hometown of Memmingen. After various reggae bands and sound projects in dub, hip-hop & drum’n’bass, MC TREASURE IRIE has been active on the mic for the Dandelion Soundsystem for over 10 years, as well as with Eisbach Callin’, Siren Sisters, at Rootsbase/Fusion Festival, and the Siko Munich. He writes his lyrics in English, patois slang, as well as in German.',
```

File: src/app/about/page.tsx (hunk 2 above)
Comment (jfcurran) [inline src/app/about/page.tsx:56]:
```suggestion
              ! We don’t record much, but when we do, it’s usually pretty good.
```

**Review by jfcurran — CHANGES_REQUESTED** [review]
caught ESLINT lacking because we used single quotes ' instead of apostrophe ’  - should be fixed with the changes.

**Review by jfcurran — APPROVED** [review]
image

**Comment by vinckr** [comment]
Thx for fix & review :heart:

**Pull request merged**